charts/seed-monitoring/charts/grafana/dashboards/owners/coredns-dashboard.json

```diff
@@ -600,6 +600,79 @@
         "alignLevel": null
       }
     },
+    {
+      "cards": {
+        "cardPadding": null,
+        "cardRound": null
+      },
+      "color": {
+        "cardColor": "#b4ff00",
+        "colorScale": "sqrt",
+        "colorScheme": "interpolateOranges",
+        "exponent": 0.5,
+        "mode": "spectrum"
+      },
+      "dataFormat": "tsbuckets",
+      "datasource": "prometheus",
+      "description": "DNS Requests Latency in seconds",
+      "fieldConfig": {
+        "defaults": {},
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 7,
+        "w": 12,
+        "x": 12,
+        "y": 15
+      },
+      "heatmap": {},
+      "hideZeroBuckets": true,
+      "highlightCards": true,
+      "id": 36,
+      "legend": {
+        "show": true
+      },
+      "links": [],
+      "pluginVersion": "7.5.10",
+      "reverseYBuckets": false,
+      "targets": [
+        {
+          "exemplar": true,
+          "expr": "sum by (le) (rate(coredns_dns_request_duration_seconds_bucket{pod=~\"$pod\"}[$__rate_interval]))",
+          "format": "heatmap",
+          "interval": "",
+          "intervalFactor": 1,
+          "legendFormat": "{{le}}",
+          "refId": "A",
+          "step": 60
+        }
+      ],
+      "timeFrom": null,
+      "timeShift": null,
+      "title": "DNS Requests Latency Heatmap (In Seconds)",
+      "tooltip": {
+        "show": true,
+        "showHistogram": true
+      },
+      "type": "heatmap",
+      "xAxis": {
+        "show": true
+      },
+      "xBucketNumber": null,
+      "xBucketSize": null,
+      "yAxis": {
+        "decimals": null,
+        "format": "short",
```

Comment:
Please consider using a matching time unit (seconds) for the y axis. Also try the decimals setting: `2` seems to work better than the default `auto`.
<img width="1477" alt="image" src="https://user-images.githubusercontent.com/23032437/144811589-6f4808f3-2b9f-41cd-8796-a851bb58d867.png">

Reply:
Thanks for the suggestion. Updated the dashboard with the recommendation.